src/Dropdown/Dropdown.test.jsx

```diff
@@ -36,6 +36,12 @@ describe('<Dropdown />', () => {
     it('with menu closed', () => {
       menuOpen(false, wrapper);
     });
+
+    it('no focus on just update', () => {
```

Comment:
name should be something like this `does not get focus when updated`

Reply:
:+1: